docker-bake.hcl

```diff
@@ -0,0 +1,58 @@
+variable "BUNDLES_OUTPUT" {
```

Comment:
@lmbarros this is required for the CI pipeline to run the builds. It's from moby/master and will be likely included in their next release (since they are also in the process of moving to GH actions).